news/2018-06-15-cwc-2018-recap-group-stage.md

```diff
@@ -0,0 +1,43 @@
+Another year of furious fruit catching competition is well and truly upon us, with the world’s best duking it out on the grandest stage of all for your viewing pleasure. Missed any of the action? Eager to see where we’re heading next? All the juicy details can be found within!
```

Comment:
missing newspost headers

Reply:
make sure they're dated for the 15th and an appropriate time in UTC as well

Reply:
Added for 1200 UTC+0, I should not PR when tired